[2024-05-10T16:42:13.490+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-10T16:42:13.559+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.task3 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-10T16:42:13.579+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.task3 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-10T16:42:13.581+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 7
[2024-05-10T16:42:13.610+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task3> on 2024-05-08 00:00:00+00:00
[2024-05-10T16:42:13.630+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'task3', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpduxp8fdp']
[2024-05-10T16:42:13.640+0000] {standard_task_runner.py:91} INFO - Job 19: Subtask task3
[2024-05-10T16:42:13.641+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=734) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-10T16:42:13.644+0000] {standard_task_runner.py:63} INFO - Started process 736 to run task
[2024-05-10T16:42:14.084+0000] {task_command.py:426} INFO - Running <TaskInstance: my_first_dag_v3.task3 scheduled__2024-05-08T00:00:00+00:00 [running]> on host 48bfb4afd9cd
[2024-05-10T16:42:14.509+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='edo' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='task3' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-10T16:42:14.513+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-10T16:42:14.546+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-10T16:42:14.550+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo third one matey']
[2024-05-10T16:42:14.576+0000] {subprocess.py:86} INFO - Output:
[2024-05-10T16:42:14.579+0000] {subprocess.py:93} INFO - third one matey
[2024-05-10T16:42:14.582+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-10T16:42:14.583+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-10T16:42:14.638+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=task3, run_id=scheduled__2024-05-08T00:00:00+00:00, execution_date=20240508T000000, start_date=20240510T164213, end_date=20240510T164214
[2024-05-10T16:42:14.696+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-10T16:42:14.730+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
